### DISCUSSION TITLE
使用 1.8.8 MiraiMC 在 Paper 1.20.1-169 上运行，不能正确收发信息

### DISCUSSION THREAD

**Discussion by BiDuang** [DreamVoid/MiraiMC#447] [问答 / Q&A]
尝试在群内发送消息时，MiraiMC 在控制台上打印了如下输出：
```
[MiraiMC] [GroupMessage/***] [***] BiDuang(***) -> null
```

而尝试使用 Chat2QQ 插件通过 MiraiMC 发送消息，似乎也不成功。

登录方式是外置的MCL通过HTTPAPI连接。

另：能否提供自动通过 HTTPAPI 登录的方法？插件完全载入后手动登录不太方便，谢谢！

祝好。

**Comment by DreamVoid** [comment]
https://github.com/DreamVoid/MiraiMC/issues/370#issuecomment-1464861351
